draft-ietf-quic-recovery.md

```diff
@@ -1031,9 +1031,8 @@ OnAckReceived(ack, pn_space):
     OnPacketAcked(acked_packet.packet_number, pn_space)
 
   DetectLostPackets(pn_space)
-
+  SendPacketsAsCongestionControllerAllows()
   pto_count = 0
-
   SetLossDetectionTimer()
```

Comment:
we should delete this. OnPacketSent() already sets the timer, and SendPacketsAsCongestionControllerAllows may not actually send a packet.

Reply:
The ack may update the RTT, so I think we should  keep this unless we add a return value to SendPacketsAsCongestionControllerAllows() indicating whether a packet was sent?

Reply:
Ah, yes, we might have to reset the timer to reflect the new RTT. But then shouldn't the RTT update code go above this?

Reply:
It's above on line 1024.

Reply:
My mistake. I saw the UpdateRTT definition below and got confused.